File: code-examples/scala/collation/collection-specify-collation.scala
/*
  Retrieve documents that match "photograph" in the "type" field,
  sorted by the Iceland collation and uppercase precedence.
*/
// start specified collation
myColl.find(Filters.eq("type", "photograph")).collation(Collation.builder().locale("is").caseLevel(true).collationCaseFirst(CollationCaseFirst.UPPER).build())
// end specified collation

File: code-examples/java-rs/collation/collection-specify-collation.java
MongoCollection<Document> myColl;

FindPublisher<Document> findPublisher = myColl.find(Filters.eq("type", "photograph"))
    .collation(Collation.builder()
        .locale("is")
        .caseFirst(CollationCaseFirst.UPPER)
        .build()
    );
Mono.from(findPublisher).subscribe();
